service/items.go

```diff
@@ -6,54 +6,51 @@ import (
 	"net/http"
 	"strconv"
 
-	"github.com/rodrigomkd/go-rest-api/common"
-	"github.com/rodrigomkd/go-rest-api/config"
+	"github.com/rodrigomkd/go-rest-api/model"
+	"github.com/rodrigomkd/go-rest-api/service/api"
+	"github.com/rodrigomkd/go-rest-api/service/csv"
 )
 
-type item struct {
-	ID          int    `json:"ID"`
-	Description string `json:"Description"`
+//struct
+type Service struct {
+	cs         csv.CSVService
+	api        api.ApiService
+	dataSource string
```

Comment:
I'm guessing this datasource is the csvfile path, it would be better to move it as an attribute of the csv service